extensions/amp-subscriptions/0.1/test/test-amp-subscriptions.js

```diff
@@ -215,39 +215,6 @@ describes.fakeWin('AmpSubscriptions', {amp: true}, (env) => {
       expect(processStateStub).to.be.calledWith(true);
     });
 
-    it(
```

Comment:
Since the `!this.doesViewerProvideAuth_` check was removed this test is no longer necessary